core/modules/czar/CzarConfig.cc

```diff
@@ -68,7 +68,9 @@ CzarConfig::CzarConfig(util::ConfigStore const& configStore)
     : _mySqlResultConfig(configStore.get("resultdb.user", "qsmaster"),
             configStore.getRequired("resultdb.passwd"),
             configStore.getRequired("resultdb.host"), configStore.getInt("resultdb.port"),
-            configStore.getRequired("resultdb.unix_socket"), configStore.get("resultdb.db","qservResult")),
+            configStore.getRequired("resultdb.unix_socket"),
+            configStore.get("resultdb.db","qservResult"),
+            configStore.getInt("resultdb.maxtablesize", 5001)),
```

Comment:
The 5001 was arbitrary so I could tell the difference between using this default value and the other and pin down where the configuration value was failing to be passed.  In practice, the difference between 5001 and 5000 is not going to make a significant difference in what happens when the software runs. That being said, I'll probably switch it back to 5000.